### DISCUSSION TITLE
apache proxy issue

### DISCUSSION THREAD

**Discussion by ufukayyildiz** [requarks/wiki#3218] [Help / Questions] [answered]
Hi,

I run wiki.js on apache server. Without ssl there is no problem. but with SSL i have issue.

I already installed lets SSL on apache. Tried to proxypass but no success :(

Should i set SSL to false in config.yml ?

**Comment by NGPixel** [comment, marked as answer]
You're applying SSL termination twice... Apache is already handling the SSL so Wiki.js shouldn't have SSL enabled.

**Reply by ufukayyildiz** [reply]
Thank you. I will try again.

**Reply by ufukayyildiz** [reply]
But when i set;

**ssl:
  enabled: false
  port: 8443**

**it refuse the connection.**

Should not i set a port for SSL?

What do i do wrong?

**Reply by NGPixel** [reply]
Like I said, you should not use SSL or the SSL port on the Wiki.js side. Use the standard port (3000 or 80) when pointing to Wiki.js.
